src/main/docker/jenkins/jenkins.yml

```diff
@@ -0,0 +1,33 @@
+# ----------------------------------------------------------------------------------------------------------------------
```

Comment:
This Directory `src/main/docker/jenkins/` also contains `swift/Dockerfile`.
All builds run in separate Docker containers, so that is no longer needed and should be removed.

Reply:
So I just deleted the `swift/Dockerfile` and one reference to it in the docs. That's how this is supposed?